lib/unifile-fs.js

```diff
@@ -32,6 +33,19 @@ class FsConnector {
     this.name = NAME;
   }
 
+  resolvePath(path) {
+    // is the path alreay resolved
+    if(path.startsWith(Os.homedir()) && Path.resolve(path).startsWith(Os.homedir()))
+      return path;
+    // or we resolve it in the user ~
+    const secure = Path.resolve(Os.homedir(), path);
```

Comment:
I understand the concern here but I don't think it should be Unifile who restricts access. 

I see two options here:
* no control, the program using the library will set the limits 
* a 'safe path' given as an argument to the constructor of FS connector. 

In the exemple of Silex, I don't think you want user to meddle in Silex home directory 😊